services/notifications.go

```diff
@@ -1149,7 +1149,7 @@ func sendDiscordNotifications(useDB *sqlx.DB) error {
 }
 
 func getUrlPart(validatorIndex uint64) string {
-	return fmt.Sprintf(` For more information visit: https://%s/validator/%v.`, utils.Config.Frontend.SiteDomain, validatorIndex)
+	return fmt.Sprintf(` For more information visit: <a href='https://%s/validator/%v'>https://%s/validator/%v</a>.`, utils.Config.Frontend.SiteDomain, validatorIndex, utils.Config.Frontend.SiteDomain, validatorIndex)
```

Comment:
Have you tested this? There might be similar problems regarding link-clickability here:
https://github.com/gobitfly/eth2-beaconchain-explorer/blob/cc4f1444b7c6d58af067a55bd7d9b4c89b161f85/services/notifications.go#L2594
https://github.com/gobitfly/eth2-beaconchain-explorer/blob/cc4f1444b7c6d58af067a55bd7d9b4c89b161f85/services/notifications.go#L2607
https://github.com/gobitfly/eth2-beaconchain-explorer/blob/cc4f1444b7c6d58af067a55bd7d9b4c89b161f85/services/notifications.go#L1849

(list may be incomplete)

Sorry, I don't know the subscription system oh too well.

Reply:
1st: You might be right, but the function is used for Discord as well. So adding a <a href= will look weird in Discord

2nd: this looks like Discord/Jira/whatever notation, not HTML. [name](url) is a pattern there, similar to <a href=url>name</a>

3rd: Similar to 2nd, just 3 times,